account_edi_simple_pdf/models/account_invoice_import_simple_pdf_mixin.py

```diff
@@ -0,0 +1,377 @@
+# Copyright 2021 Akretion France (http://www.akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+import shutil
+import subprocess
+from tempfile import NamedTemporaryFile
+
+from odoo import _, api, models
+from odoo.exceptions import UserError
+from odoo.osv import expression
+
+logger = logging.getLogger(__name__)
+try:
+    import fitz
+except ImportError:
+    logger.debug("Cannot import PyMuPDF")
+try:
+    import regex
+except ImportError:
+    logger.debug("Cannot import regex")
+try:
+    import pdftotext
+except ImportError:
+    logger.debug("Cannot import pdftotext")
+try:
+    import pypdf
+except ImportError:
+    logger.debug("Cannot import pypdf")
+
+
+class AccountInvoiceImportSimplePdfMixin(models.AbstractModel):
```

Comment:
WARNING odoo odoo.models: The model account.invoice.import.simple.pdf.mixin has no _description